typings/audit.d.ts

```diff
@@ -16,21 +16,22 @@ declare global {
       scoreMedian: number;
     }
 
-    export interface ScoringModes {
+    export interface ScoreDisplayModes {
       NUMERIC: 'numeric';
       BINARY: 'binary';
+      MANUAL: 'manual';
+      INFORMATIVE: 'informative';
+      NOT_APPLICABLE: 'not-applicable';
     }
 
-    export type ScoringModeValue = Audit.ScoringModes[keyof Audit.ScoringModes];
+    export type ScoringModeValue = Audit.ScoreDisplayModes[keyof Audit.ScoreDisplayModes];
```

Comment:
should this type be `ScoreDisplayMode` now? Seems like it makes sense

Reply:
yeah sg